#exception handling/error handling
try:
    print("hello world"+10)
except:
    print("hello world!")
